test/unit/calculators/coronavirus_find_support_calculator_test.rb

```diff
@@ -380,6 +385,11 @@ class CoronavirusFindSupportCalculatorTest < ActiveSupport::TestCase
 
     context "#next_question" do
       context "user is on the need_help_with? node" do
+        should "return feel_safe? when 'none' has been chosen" do
```

Comment:
Pedantic but `should "return feel_safe when ....` no question mark now on the question

Reply:
Thanks @reggieb Umm - there are quite a few of these.  I'll create a new PR for their removal rather than do that here.